src/mono/wasm/runtime/corebindings.c

```diff
@@ -42,6 +42,10 @@ extern void mono_wasm_typed_array_from_ref (int ptr, int begin, int end, int byt
 extern void* mono_wasm_invoke_js_blazor (MonoString **exceptionMessage, void *callInfo, void* arg0, void* arg1, void* arg2);
 #endif /* ENABLE_LEGACY_JS_INTEROP */
 
+// HybridGlobalization
+extern void mono_wasm_change_case_invariant(MonoString **exceptionMessage, const uint16_t* src, int32_t srcLength, uint16_t* dst, int32_t dstLength, mono_bool bToUpper);
```

Comment:
is mono_bool guaranteed to be a 32-bit int?

Reply:
Yes, it is an alias for ` int32_t:
https://github.com/dotnet/runtime/blob/afcfc4be7fcaf0a00f37c7d5ec9b793a5b8e43f0/src/native/public/mono/utils/details/mono-publib-types.h#L80